src/Modules/Guilds/Hooks/useProposal.ts

```diff
@@ -26,7 +27,10 @@ export const formatterMiddleware = (data: InitialProposal): Proposal => {
   // Add timeDetail
   const currentTime = moment();
   let differenceInMilliseconds = currentTime.diff(clone.endTime);
-  let timeDifference = moment.duration(differenceInMilliseconds).humanize();
+  let timeDifference =

```

Comment:
💡 If you were to move the logic in `TimeDetail`to its own file and logic you could re-use it here instead.